dlt/common/destination/client.py

```diff
@@ -150,6 +150,9 @@ class DestinationClientConfiguration(BaseConfiguration):
     credentials: Optional[CredentialsConfiguration] = None
     destination_name: Optional[str] = None  # name of the destination
     environment: Optional[str] = None
+    info_tables_query_threshold: int = dataclasses.field(
```

Comment:
you can just set `1000` directly. it is immutable so no need to use a factory. also pls move it to `DestinationClientDwhConfiguration` - not all destinations are SQL based, not all need this config